pyls/config/pylint_conf.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Palantir Technologies, Inc.
+import logging
+import os
+from pyls._utils import find_parents
+from .source import ConfigSource, _get_opt, _set_opt
+
+log = logging.getLogger(__name__)
+
+PROJECT_CONFIGS = ['.pylintrc', 'pylintrc']
+
+CONFIG_KEYS = {  # 'option': 'section key'
+    'disable': 'MESSAGES CONTROL',
+    'ignore': 'MASTER',
+    'max-line-length': 'FORMAT',
+}
```

Comment:
@goanpeca @youben11 From all of pylint's configuration options, we only extract three fields and when the document is passed to pylint, pylintrc is ignored.

- Why is there a need to extract configuration at all?
- Why are we not calling pylint in a way that it itself looks for its configuration?